apps/docs/stories/components/payments-list.stories.tsx

```diff
@@ -24,6 +24,7 @@ const meta: Meta = {
   },
   decorators: [
     customStoryDecorator,
+    // @ts-ignore - Ignore Storybook bug (reference to bug issue)
```

Comment:
Nice! Is it possible to do at a more global level?

Reply:
I think so, but I am not sure if that would be good. It would bypass all "type mismatch" errors, which is quite common.

Reply:
Ahh, I see. Good call 👍